src/main/java/frc/robot/RobotContainer.java

```diff
@@ -26,7 +26,7 @@ public class RobotContainer {
   private final CommandXboxController driver =
       new CommandXboxController(OperatorConstants.driverControllerPort);
   public final Drivetrain drivetrain = new Drivetrain();
-  private final ArmSubsystem armSubsystem = new ArmSubsystem();
+  //private final ArmSubsystem armSubsystem = new ArmSubsystem();
```

Comment:
Resolve bug with #42 instead of just commenting out the code that causes the error.